python_modules/jupedsim/tests/test_distributions.py

```diff
@@ -204,6 +207,10 @@ def test_distribution_in_circle_by_number_creates_correct_points():
             j = j + 1
 
 
+def foo(*, a, b):
+    return a + b
+
+
```

Comment:
I guess this was just a test function?
Please remove it

Reply:
removed